permafrost/templates/permafrost/permafrostrole_form.html

```diff
@@ -103,8 +123,12 @@ <h6 class="m-0 text-primary font-weight-bold">
                 </div>
             </div>
             <div class="form-row border-bottom-gray-300">
-                <p class="col-6 font-weight-bold">{% trans 'Required Permissions' %}</p>
-                <p class="col-6 font-weight-bold">{% trans 'Optional Permissions' %}</p>
+                <p class="col-4 font-weight-bold">{% trans 'Required Permissions' %}</p>
+                <p class="col-4 font-weight-bold">{% trans 'Optional Permissions' %}</p>
+                {% if object and not object.is_default_role %}
+                <a class="ml-auto text-success font-weight-bold text-right" data-toggle="modal" data-target="#form_modal" href="{% url 'permafrost:custom-role-add-permissions' slug=object.slug %}"
```

Comment:
double-check bootstrap 5 text alignment utilities:  https://getbootstrap.com/docs/5.0/utilities/text/#text-alignment

Reply:
instead of `text-left` its `text-start`